services/notifications/src/simcore_service_notifications/clients/postgres/_liveness.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from asyncio import Task
+from datetime import timedelta
+
+from fastapi import FastAPI
+from models_library.healthchecks import IsResponsive, LivenessResult
+from servicelib.async_utils import cancel_wait_task
+from servicelib.background_task import create_periodic_task
+from servicelib.db_asyncpg_utils import check_postgres_liveness
+from servicelib.fastapi.db_asyncpg_engine import get_engine
+from servicelib.logging_utils import log_catch
+
+_logger = logging.getLogger(__name__)
+
+
+class PostgresLiveness:
+    def __init__(self, app: FastAPI) -> None:
+        self.app = app
+
+        self._liveness_result: LivenessResult = IsResponsive(elapsed=timedelta(0))
+        self._task: Task | None = None
+
+    async def _check_task(self) -> None:
+        self._liveness_result = await check_postgres_liveness(get_engine(self.app))
+
+    @property
+    def is_responsive(self) -> bool:
+        return isinstance(self._liveness_result, IsResponsive)
+
+    async def setup(self) -> None:
+        self._task = create_periodic_task(
+            self._check_task,
+            interval=timedelta(seconds=10),
```

Comment:
minor: hardcoded numbers or timings should be either collected somehwere in "constants.py" or even be env-vars, so we can act quickly if ever needed